test/lib/cql_test_env.cc

```diff
@@ -375,7 +375,6 @@ class single_node_cql_env : public cql_test_env {
             cfg->num_tokens.set(256);
             cfg->ring_delay_ms.set(500);
             auto features = cfg->experimental_features();
-            features.emplace_back(db::experimental_features_t::CDC);
```

Comment:
The downside of this change is that the full flow, *with* cdc, gets less testing.
But I don't know what to suggest.... I guess your patch makes sense.